src/benchmarklib/benchmark_utils.hpp

```diff
@@ -25,8 +25,8 @@ using TimePoint = std::chrono::high_resolution_clock::time_point;
 using NamedQuery = std::pair<std::string, std::string>;
 using NamedQueries = std::vector<NamedQuery>;
 
-using EncodingMapping = std::map<std::string, ColumnEncodingSpec>;
-using TableColumnEncodingMapping = std::unordered_map<std::string, EncodingMapping>;
+using DataTypeEncodingMapping = std::map<DataType, ColumnEncodingSpec>;
+using TableColumnEncodingMapping = std::unordered_map<std::string, std::map<std::string, ColumnEncodingSpec>>;
```

Comment:
That the inner map is a column-name->spec mapping needs doc now. And mixed map and unordered_map is inconsistent, you could fix that, too, if you want to